lib/Client.php

```diff
@@ -134,7 +132,7 @@ public function transaction() {
      */
     public function close() {
         /** @var Promise $promise */
-        $promise = all(promises($this->promisors));
+        $promise = all($this->promisors);
```

Comment:
Understood thanks.

Then it is possible to get enough of this place at all promisors promise through array_map?